CHANGELOG/unreleased/kong/11464.yaml

```diff
@@ -0,0 +1,7 @@
+message: Fix TTL not working in DB-less and Hybrid mode
```

Comment:
The message should like `Fix an issue that ...`

Reply:
done